NN-notebook-v7/notebook-v7.md

```diff
@@ -0,0 +1,118 @@
+
+There is a [community repository of unofficial Notebook extensions](https://jupyter-contrib-nbextensions.readthedocs.io), along with [corresponding documentation](https://jupyterlab-contrib.github.io/migrate_from_classical.html) indicating which of these extensions have analogs or ports in JupyterLab.
+
+### Critical extensions needed for Notebook v7
+
+The following Notebook v6 extensions have been flagged by the community as critical to several major deployments and widely-used workflows. We will work with the extension developers to help port these (or help make new equivalent extensions) for Notebook v7:
+
+- [Nbgrader](https://github.com/jupyter/nbgrader)
+- [RISE](https://github.com/damianavila/RISE)
+- [Jupytext](https://github.com/mwouts/jupytext/issues/271)
+- IPython Parallel (filebrowser tab)
+
+### Extensions whose functionality will be built into Notebook v7
+
+The following extensions will have largely equivalent experiences shipping in core Notebook v7. We will need to track whether there are any remaining differences in their functionality that are either regressions, or potentially disrupting or confusing to existing users.
+
+- [Table of Contents](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/toc2)
+- [Collapsible headings](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/collapsible_headings)
+
+## Major new features in Notebook v7
+
+In addition to having parity with the Notebook v6 user experience and widely-used extensions, Notebook v7 will inherit major new features that have been developed in the JupyterLab project, thus providing automatically a number of improvements to Notebook users as they transition to v7:
+
+- Debugger
+- Real-time collaboration
+- Theming and dark mode
+- Internationalization
+- Improved Accessibility
+- Support for many JupyterLab extensions
+
+## Technical details
+
+- [RetroLab](https://github.com/jupyterlab/retrolab) running on [Jupyter Server](https://github.com/jupyter-server/jupyter_server) will be the basis for Notebook v7.
+- The current [RetroLab](https://github.com/jupyterlab/retrolab) commit history will be grafted into the [Jupyter Notebook repository](https://github.com/jupyter/notebook) in a pull request.
+- Notebook v7 will continue to be released as the `notebook` Python package.
+- Issues and PRs in the [Notebook repo](https://github.com/jupyter/notebook) will be triaged in this transition.
+
+## Timeline
+
+We anticipate that Notebook v7 will be released in 2022.
```

Comment:
Should we mention we would like to aim for a first pre-release for the new year?

Reply:
That may be a good way to convey that we want to do this sooner than later.

Reply:
```suggestion
We anticipate that Notebook v7 will be released in early 2022.
```